plugins/com.yakindu.solidity.tests/META-INF/MANIFEST.MF

```diff
@@ -7,14 +7,14 @@ Bundle-SymbolicName: com.yakindu.solidity.tests;singleton:=true
 Bundle-ActivationPolicy: lazy
 Require-Bundle: com.yakindu.solidity,
  org.eclipse.xtext.testing,
- org.eclipse.xtext.xbase.testing,
- org.eclipse.xtext.xbase.lib,
  org.junit.platform.commons,
  org.junit.platform.launcher,
  org.junit.platform.engine,
  org.junit.jupiter.engine,
  org.opentest4j,
- org.junit.jupiter.params
+ org.junit.jupiter.params,
+ org.eclipse.xtext.xbase.lib;bundle-version="2.14.0",
```

Comment:
Is the explicit version required here?

Reply:
no - tests will be reworked in another PR later, they are not even build currently